dispatcher/frontend-ui/src/components/ScheduleEditor.vue

```diff
@@ -377,8 +377,13 @@
         let parent = this;
 
         ["name", "category", "enabled", "periodicity"].forEach(function (key) {
-          if (parent.edit_schedule[key] != parent.schedule[key])
-            payload[key] = parent.edit_schedule[key];
+          if (parent.edit_schedule[key] != parent.schedule[key]) {
```

Comment:
Wouldn't it be easier to do `parent.edit_schedule[key].trim();` instead? We'd get the UI updated as for clone and that would remove the need for this special treatment here.